### PR TITLE
change_upload_implementation

### PR DESCRIPTION
**Pull request by bassem97**

### CODE REVIEW

File: src/UpupUploader.tsx [hunk 1]
```diff
@@ -128,7 +128,11 @@ export const UpupUploader: FC<UpupUploaderProps & RefAttributes<any>> =
         useImperativeHandle(ref, () => ({
             async uploadFiles() {
                 if (files.length === 0) return null
-                const filesList = mutatedFiles.length > 0 ? mutatedFiles : files
+                console.log(files)
```
Comment (MohamedBechirMejri) [inline src/UpupUploader.tsx]: remove console.log

File: src/UpupUploader.tsx [hunk 2]
```diff
@@ -185,13 +189,14 @@ export const UpupUploader: FC<UpupUploaderProps & RefAttributes<any>> =
...
-                                await putObject({
+                                await uploadObject({
                                     client,
                                     bucket,
                                     key,
                                     file,
                                 })
-                                    .then(() => {
+                                    .then(data => {
+                                        console.log('data', data)
```
Comment (MohamedBechirMejri) [inline src/UpupUploader.tsx]: remove console.log

**Review by MohamedBechirMejri — APPROVED** [review]
Looks Good!

**Review by bassem97 — COMMENTED** [review]
2nd verification done

**Pull request merged**